components/layout/DocsLayout.js

```diff
@@ -111,6 +112,33 @@ export default function DocsLayout({ post, navItems = {}, children }) {
                   {generateEditLink(post)}
                 </p>
               </div>
+              { post.releaseNoteLink &&
+                // show only when it is related to specification (/docs/reference/specification) AND is not a pre-release 
+                // for example, if the post's title is "3.0.0 (Pre-release)", which will not have RN, so do not render this section. 
+                <div className="w-full mt-5 py-3 px-2 text-center rounded-lg bg-secondary-100">
+                  <div>
+                    <span className="text-sm font-sans antialiased text-gray-800"> 
+                      {`What is new in v${post.title}? Have a look at the `} 
+                    </span> 
+                    <Link href={post.releaseNoteLink}>
+                      <span>
```

Comment:
please don't add a span tag just after the Link component because then, the links aren't recognizable as normal links by the browser.

Reply:
do I need to make it as div? @akshatnema

Reply:
or can i just remove span and have only the a tag?

Reply:
remove the span and only a tag should be there.